Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration-new/problem-resources/problemResourcesGridColumns.tsx

```diff
@@ -80,32 +90,6 @@ const problemResourceFilterableColumns: AdministrationGridColDef[] = [
         align: 'center',
         headerAlign: 'center',
     },
-    {
-        field: 'problemId',
-        headerName: 'Problem Id',
-        flex: 0,
-        type: 'number',
```

Comment:
Why the problemId is removed? How the resource can be tied to a specific problem when names can duplicate?